src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/None.xaml.pt-BR.xlf

```diff
@@ -1,127 +1,6 @@
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|CustomToolNamespace|DisplayName">
-        <source>Custom Tool Namespace</source>
-        <target state="translated">Namespace da Ferramenta Personalizada</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|CustomToolNamespace|Description">
-        <source>The namespace into which the output of the custom tool is placed.</source>
-        <target state="translated">O namespace em que o resultado da ferramenta personalizada é colocado.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|Identity|Description">
-        <source>The item specified in the Include attribute.</source>
-        <target state="translated">O item especificado no atributo Include.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|FullPath|DisplayName">
-        <source>Full Path</source>
-        <target state="translated">Caminho Completo</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|FullPath|Description">
-        <source>Location of the file.</source>
-        <target state="translated">Localização do arquivo.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|FileNameAndExtension|DisplayName">
-        <source>File Name</source>
-        <target state="translated">Nome do Arquivo</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|FileNameAndExtension|Description">
-        <source>Name of the file or folder.</source>
-        <target state="translated">Nome do arquivo ou da pasta.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|LastGenOutput|Description">
-        <source>The filename of the last file generated as a result of the SFG.</source>
-        <target state="translated">O nome do último arquivo gerado como resultado do SFG.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="BoolProperty|DesignTime|Description">
-        <source>A value indicating whether this file has a designer.</source>
-        <target state="translated">Um valor indicando se esse arquivo tem um designer.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="BoolProperty|AutoGen|Description">
-        <source>A value indicating whether this is a generated file.</source>
-        <target state="translated">Um valor indicando se esse é um arquivo gerado.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="StringProperty|CustomTool|Description">
-        <source>DTE Property for accessing the Generator property.</source>
-        <target state="translated">Propriedade DTE para acessar a propriedade Generator.</target>
-        <note />
-      </trans-unit>
-    </body>
+    <body />
```

Comment:
I think we can now opt these out of localization - but I'm not sure how.

Reply:
Which will also involve removing the files.

Reply:
The file is empty, so certainly nobody will bother localizing anything in them, but not having localization files would be nicer. Will add a task for it